From 33bef6a2dc38022fd4f9e216b22cffb5ac4bbd97 Mon Sep 17 00:00:00 2001
From: Mister Oyster <oysterized@gmail.com>
Date: Sun, 20 Aug 2017 17:33:58 +0200
Subject: [PATCH] vendor: lineage: unbreak mtk builds

---
 build/core/mtk_target.mk | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/build/core/mtk_target.mk b/build/core/mtk_target.mk
index 11a7f8c..664c6aa 100644
--- a/build/core/mtk_target.mk
+++ b/build/core/mtk_target.mk
@@ -1,16 +1,16 @@
 # Bring in Mediatek helper macros
-include vendor/lineage/build/core/mtk_utils.mk
+#include vendor/lineage/build/core/mtk_utils.mk
 
-ifeq ($(BOARD_USES_MTK_HARDWARE),true)
-    mtk_flags := -DMTK_HARDWARE
-
-    TARGET_GLOBAL_CFLAGS += $(mtk_flags)
-    TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
-    CLANG_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
-    CLANG_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
-
-    2ND_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
-    2ND_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
-    2ND_CLANG_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
-    2ND_CLANG_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
-endif
+#ifeq ($(BOARD_USES_MTK_HARDWARE),true)
+#    mtk_flags := -DMTK_HARDWARE
+#
+#    TARGET_GLOBAL_CFLAGS += $(mtk_flags)
+#    TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
+#    CLANG_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
+#    CLANG_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
+#
+#    2ND_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
+#    2ND_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
+#    2ND_CLANG_TARGET_GLOBAL_CFLAGS += $(mtk_flags)
+#    2ND_CLANG_TARGET_GLOBAL_CPPFLAGS += $(mtk_flags)
+#endif
-- 
2.11.0

